spec.txt

```diff
@@ -7493,10 +7485,14 @@ Although [link titles] may span multiple lines, they may not contain
 a [blank line].
 
 An [inline link](@) consists of a [link text] followed immediately
-by a left parenthesis `(`, optional [whitespace], an optional
-[link destination], an optional [link title] separated from the link
-destination by [whitespace], optional [whitespace], and a right
-parenthesis `)`. The link's text consists of the inlines contained
+by a left parenthesis `(`, optional spaces, tabs, and up to one line ending,
+optional link information, and a right parenthesis `)`.
+Link information consists of a [link destination], an optional [link title], and
+optional spaces, tabs, and up to one line ending.
+If a title is present, it must be separated from the link destination by spaces,
+tabs, and up to one line ending.
```

Comment:
@jgm How about this? It introduces “link information” as a term, which I hope fixes your [comments here](https://github.com/commonmark/commonmark-spec/pull/618#discussion_r339890394).

Introducing this term may also make it possible to use it in definitions, and images (although then the term may be better as “resource information”).

Term could be something different though!

Reply:
Nice!